bigtable/client/table.cc

```diff
@@ -191,5 +191,36 @@ bool Table::CheckAndMutateRow(std::string row_key, Filter filter,
   return response.predicate_matched();
 }
 
+/**
+ * Send request ReadModifyWriteRowRequest to modify the row and get it back
+ */
+Row Table::CallReadModifyWriteRowRequest(
+    btproto::ReadModifyWriteRowRequest row_request) {
+  auto error_message =
+      "ReadModifyWriteRowRequest(" + row_request.table_name() + ")";
+  auto response_row = RpcUtils::CallWithoutRetry(
+      *client_, rpc_retry_policy_->clone(), &StubType::ReadModifyWriteRow,
+      row_request, error_message.c_str());
+
+  std::vector<bigtable::Cell> cells;
+  for (auto& family : response_row.row().families()) {
+    for (auto& column : family.columns()) {
+      for (auto cell : column.cells()) {
+        std::vector<std::string> labels;
+        for (auto label : cell.labels()) {
+          labels.emplace_back(label.c_str());
+        }
+        bigtable::Cell new_cell(response_row.row().key(), family.name(),
```

Comment:
This is copying all the data, is there a way to move the data out of the response into the `new_cell`? For example, the code around [here](https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/master/bigtable/client/internal/readrowsparser.cc#L53) does that.